### ISSUE TITLE
MQF2DistributionLoss and MultivariateNormalDistributionLoss not available anymore

### ISSUE THREAD

**Issue by CapussoM** [jdb78/pytorch-forecasting#1052]
Hello! I was trying to train DeepAR and NHiTS, the problem is that the two losses used in those models seem to not be available anymore in the library, where can I find them?

I tried both: 
`from pytorch_forecasting.metrics`

and
`from pytorch_forecasting.metrics.distributions`

but in the first it's not present and the second obviously throws an error since metrics is not a package. Are those two losses still available somewhere?

**Comment by why362** [comment]
> 

i encountered the same problem, do you have a solution yet?

**Comment by erykml** [comment]
I did not dive that deep into the issue, but I also encountered it while using Google Colab. 
Then, I installed the library locally, and the issue was gone. 
Also, you can see here that the loss functions are not actually gone from the codebase: https://github.com/jdb78/pytorch-forecasting/blob/master/pytorch_forecasting/metrics/distributions.py#L43.

**Comment by RenaissanceEngineering** [comment]
Still not working on Google Colab
